simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/StationState.java

```diff
@@ -65,8 +65,8 @@ public boolean isCharging(Integer evseId) {
         return findEvse(evseId).isCharging();
     }
 
-    public CableStatus getCableStatus(int connectorId) {
-        return findConnector(connectorId).getCableStatus();
+    public CableStatus getCableStatus(int evseId, int connectorId) {
```

Comment:
Consider removing this method. As you already have Evse instance fetched using evseId in the class which calls getCableStatus